app/views/dashboard/locations/show.html.erb

```diff
@@ -1 +1,25 @@
-<%= render "shared/location" %>
\ No newline at end of file
+<turbo-frame id="location_view">
+  <div class="w-full h-full p-8 flex flex-col">
+    <div class="relative w-full">
+      <div class="flex flex-row justify-between mb-8">
+        <button class="text-left pointer text-sm underline py-1">
+          <%= link_to "Edit", edit_dashboard_map_location_path(map_id: @map.id, id: @location.id), target: '_top' %>
+        </button>
```

Comment:
Don't you think it's better to just format the `<a>` as a button instead of wrapping it in a button?